### DISCUSSION TITLE
RabbitMQ Installation and Configuration issue

### DISCUSSION THREAD

**Discussion by dhrbhat1** [rabbitmq/rabbitmq-server#9754] [Other]
### Describe the bug

We've integrated Drillbench v2022.2.2 into the sequence and executed the scripted package installation on Citrix VDI in System context with the following steps:

1. Created a System environment variable: RABBITMQ_BASE=C:\ProgramData\RabbitMQ
2. Ran Erlang OTP Version 25.1.2
3. Ran RabbitMQ Version 3.11.2 after a 5-minute wait.

However, we were unable to achieve RabbitMQ service in a running state (screenshot attached) We have tried all the steps of running the rabbitmq in a secured way.
![image](https://github.com/rabbitmq/rabbitmq-server/assets/105770370/c5819fd0-d538-4382-931e-27c455d30c12)

### Reproduction steps

1. NA
2.
3.
...

### Expected behavior

Drillbench requires Rabbit mq and erlang to be installed. We want all the 3 components to be installed in a sequence and smoothly.

### Additional context

This task is bit urgent becuase we want to deploy Drillbench in production as soon as possible.

**Comment by michaelklishin** [comment]
Please use Discussions for questions in the future.

@dhrbhat1 we cannot suggest much with a single screenshot that only reports the non-zero exit code.

See Windows service logs and [RabbitMQ node logs](https://rabbitmq.com/logging.html) (if they exist), then go through the [list of most common pitfalls on Windows](https://rabbitmq.com/windows-quirks.html).
